theme/less/_navigation.less

```diff
@@ -93,13 +95,16 @@
 #main-navigation li.has-children > a:after
 {
   content: url('/images/arrow_bullet.png');
-  padding-left: 10px;
+  padding-left: 11px;
+  vertical-align: sub;
 }
 
-#main-navigation ul ul li.has-children > a:after
+#main-navigation ul ul li.has-children a:after
 {
   position: absolute;
   right: 10px;
+  top: 33%;
+  padding-right: 10px;
```

Comment:
Maybe it's right to do here `padding: 0 10px 0 0`.

Reply:
Same as here https://github.com/browser-automation/cba-website/pull/31#discussion_r465275029